_i18n/es/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,53 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-## Operating Systems:  Windows, Linux, Mac
+{% include disclaimer.html translated="true" version=page.version %}
+## Sistemas Operativos:  Windows, Linux, Mac
 
-- Retrieve your 25 word @mnemonic-seed that you saved when creating your old Monero @wallet
+- Recupera la semilla mnemónica de 25 palabras que guardaste cuando creaste tu viejo monedero de Monero.
 
-### Account Software:  monero-wallet-cli
+### Software de cuenta: monero-wallet-cli
 
-- Open a command prompt and navigate to the drive and directory that contains monero-wallet-cli
+- Abre una consola de comandos y navega al disco y directorio que contiene monero-wallet-cli.
 
-- At the command prompt type:  `monero-wallet-cli --restore-deterministic-wallet`
+- En la consola, escribe: `monero-wallet-cli --restore-deterministic-wallet`
 
-- Once you press enter you will be prompted for a wallet file name.  Give your wallet a new name, any name will do
+- Una vez que presiones Enter se preguntará por un nuevo nombre de monedero. Escribe cualquier nombre, cualquiera funcionará.
 
-- Press enter again and you will be prompted for a password.  Give your wallet a new and long password
+- Presiona Enter de nuevo y se pedirá una contraseña. Da a tu monedero una contraseña una larga y nueva contraseña.
 
-- Press enter again you will be prompted to repeat the password
+- Presiona Enter de nuevo y se preguntará por repetir la contraseña.
 
-- Press enter again and you will be prompted for the 25 word electrum style mnemonic seed that you retrieved earlier
+- Presiona Enter nuevamente y se pedirá la semilla mnemónica de 25 palabras estilo electrum que recuperaste antes.
 
--  You will then be prompted with "Restore from specific blockchain height (optional, default 0):" Default will start the restore process from the beginning of the Monero blockchain. If you don't know the specific blockchain height, just hit enter. (Specifying a specific blockchain height will start the restore process from that specific height. This will save a bit of time in scanning, if you know what starting blockchain height your initial funds were transacted for this specific account.)
+- Se pedirá ahora el restaurar desde una altura específica en la blockchain (Restore from specific blockchain height), opcional, default 0: Default empezará el proceso de restauración desde el inicio de la blockchain de Monero. Si no conoces la altura de bloque específica, sólo presiona Enter. (Dar una altura de bloque específica iniciará el proceso de restauración desde esa misma altura. Esto ahorrará un poco de tiempo en el escaneo, si conoces la altura de bloque en la cual tus fondos iniciales fueron tramitados para tu cuenta en específico).
```

Comment:
Is there a translation for that option? Like in the GUI.

Reply:
No, not yet. The CLI hasn't been a priority, for now. But if you check the text out, you were missing the "()" for "optional, default:0", so the meaning of the text changed roughly, even the instruction in the CLI, so that is why I suggest to keep the English text, and with a comma, mark what it means.

Reply:
I just did it because I was not sure about how to translate that option, because it is something of monero-wallet-cli. Maybe we can translate it now, or like with the moneropedia, we just left it that way for now and change it later when there's an official translation of the CLI.

Reply:
Is better to keep it as it is because there isn't any translation yet. I know I would have also translated the text because I'm sure I wouldn't have seen if there was a translation for the CLI.